lib/Sema/CSSimplify.cpp

```diff
@@ -14891,6 +14866,52 @@ ConstraintSystem::simplifyRestrictedConstraintImpl(
         {getConstraintLocator(locator), restriction});
     return SolutionKind::Solved;
   }
+
+  // [Element] ===> InlineArray<N, Element>
+  case ConversionRestrictionKind::ArrayLiteralToInlineArray: {
```

Comment:
Should this result in a score increase for `SK_ImplicitValueConversion`?

Wouldn't situations like:
```
func test(_: [Int]) {}
func test(_: Slab<Int, 2>) {}

test([1, 2])
```

Be ambiguous otherwise?

Reply:
Might be worth adding a test-case like that to `slab.swift` since I don't see any that deal with ambiguities...